crates/web5/src/credentials/verifiable_credential_1_1.rs

```diff
@@ -538,6 +584,33 @@ impl core::fmt::Debug for JoseVerifier {
     }
 }
 
+#[derive(Clone)]
+struct JoseVerifierAlwaysTrue {
```

Comment:
What is this for?

Reply:
As in, why have a verifier that always returns true?

Reply:
because josekit constraints

josekit offers a `decode_unsecured` function, but it fails if the jose header includes an `alg`, which ours will in this case. so the alternative would be to decode the header, then mutate it, then create a new jwt token, then pass it into `decode_unsecured`. this just seemed easier. josekit is great, but it does offer a sort of esoteric API surface. 

if you have alternatives then I'm game!